README.md

```diff
@@ -1 +1,111 @@
+
+As programmers, we strive to write code that models the real-world. The programs we write serve a purpose––whether you're creating a simple command line game or an app to help hospitals manage patient data, our code is designed to do a real job, like run a game or communicate critical information. 
+
+### Where Will You Find Nested Hashes?
+
+Nested hashes are a very common way to store and operate on complex associated data in a program. You are likely to encounter them any time you find yourself working with a large collection of information. In particular, you will encounter these data structures when working with data you will pull from APIs. 
+
+API stands for "Application Programming Interface" and here refers to the way in which organizations, companies and governments will expose their data to the public for use. 
+
+New York City, for example, has a robust API called NYC Open Data. Developers can connect to this API to find information about city programs, public housing, parks, schools, construction, you name it. 
+
+When you send a request for data to such an API, the data you get back from them will be in the form of a nested hash that can contain information about thousands of records. 
+
+We'll learn a lot more about APIs later on in this course. For now, just understand that nested hashes are a very common occurrence in programming, they are used to store complex collections of data and you will encounter them in particular when working with APIs, among other places.
+
+### Nested Hashes Model Real-World Data
+
+We can imagine so many real-world situations and environments in which we are dealing with complicated collections of data. Take the example of a simple address book. 
+
+An individual contact from address book can be stored in a simple hash:
+
+```ruby
+zoe = {name: "Zoe", phone: "917-555-9875", address: "24 W. 124th Street, NY, NY"}
+``` 
+
+However, an address book contains more than one contact. We could collect all of our individual person hashes and store them in a `contacts` array:
+
+```ruby
+contacts = [
+	{last_name: "DeBenedetto", first_name: "Zoe", phone: "917-555-9875"}, 
+	{last_name: "DeBenedetto", first_name: "Richard", phone: "917-555-0912"}
+	{last_name: "Londin", first_name: "Victoria", phone: "917-555-0912"}
+]
+```
+
+This works, but it's not a great model of a real address book. In a real address book, contacts are sorted, or grouped, usually by last name. In our collection of hashes above, there is no way to organize our data like that. 
+
+This is where nested hashes come in. 
+
+## Our First Nested Hash
+
+With a nested hash, we can create a key that points to a value of *another hash or an array*. Let's take a look:
```

Comment:
Mention that remember anything can be a value in a hash. So! hashes are included in anything :)